workflows/argo/offline-diags.yaml

```diff
@@ -43,6 +43,8 @@ spec:
             value: "{{inputs.parameters.wandb-project}}"
           - name: WANDB_RUN_GROUP
             value: "{{inputs.parameters.wandb-group}}"
+          - name: WANDB_JOB_TYPE
+            value: offline_report
```

Comment:
chosen to be consistent with the job_type defined in `StepMetadata` in offline report code.